source/runbooks/rotating-secrets.html.md.erb

```diff
@@ -13,13 +13,31 @@ We don't have many secrets on stored on the Modernisation Platform, but they are
-| GitHub MP CI User Password | Used to log in and set the PAT | AWS Secrets Manager | Log in to GitHub as the user and reset the password, update the secret |
-| ModernisationPlatformOrganisationManagement IAM user in MoJ root account | Used to perform limited activities in the root account. No longer used as replaced by OIDC but user kept for breakglass purposes. | Not stored | No active access keys, if keys or password needed contact Operations Engineering |
-| Modernisation Platform Account Root User Password | Only used during initial platform set up, log in prevented via SCP and no password or keys set | Not stored | Disable or move account to a non SCP protected OU and follow the password reset steps |
+| PagerDuty Token | pagerduty_token | Used by PagerDuty Terraform to manage PagerDuty resources|AWS Secrets Manager | Contact Operations Engineering to issue a new token and update the secret. |
+| PagerDuty Modernisation Platform Team user | pagerduty_integration_keys | Used for dead-end notifications as all schedules need a user | Not stored | Use password reset process if needed |
+| Slack Webhook URL | slack_webhook_url | Used to post alarms to Slack | AWS Secrets Manager | Contact Operations Engineeering to issue a new incoming webhook for the `Modernisation Platform Alerts` custom Slack application.  Revoke the old incoming webhook and update the secret. |
+| GitHub MP CI User PAT | github_ci_user_pat | Used to create PRs etc in GitHub actions and deploy GitHub resources via Terraform | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| GitHub MP CI User Environments Repo PAT | github_ci_user_environments_repo_pat | Used in reusable pipelines of the modernisation-platform-environments repository. This is so that the CI user can post comments in PRs, e.g. tf plan/apply output. | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret.|
+| GitHub MP CI User Password | github_ci_user_password | Used to log in and set the PAT | AWS Secrets Manager | Log in to GitHub as the user and reset the password, update the secret |
+| environment_management | Environment Management | IDs for AWS-specific resources for environment management, such as organizational unit IDs | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| nuke_account_ids | Nuke ID List | Account IDs to be auto-nuked on weekly basis. CAUTION: Any account ID you add here will be automatically nuked! This secret is used by GitHub actions job nuke.yml inside the environments repo, to find the Account IDs to be nuked. |	Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
```

Comment:
These are not secrets so don't need to be mentioned here or rotated

Reply:
They are listed as secrets on the https://eu-west-2.console.aws.amazon.com/secretsmanager/listsecrets?region=eu-west-2 page. The GITHUB ones were in the documentation as is - https://user-guide.modernisation-platform.service.justice.gov.uk/runbooks/rotating-secrets.html#introduction

Reply:
Hey, so my comment is just referring to line 25, github just shows the lines above for context.  Ok we can keep them here then but these should not be rotated, also this text can be removed I think `CAUTION: Any account ID you add here will be automatically nuked!`

Reply:
Again, that was in place before I amended the document. I can remove it though.

Reply:
Removed